tests/voq/voq_helpers.py

```diff
@@ -24,7 +25,10 @@ def check_host_arp_table(host, neighbor_ip, neighbor_mac, interface, state, arpt
     """
 
     if arptable is None:
-        arptable = host.switch_arptable()['ansible_facts']
+        if host.is_multi_asic:
```

Comment:
I think you don't need this check here.
I think we can use `asic.switch_arptable()['ansible_facts']` for single and multi asic line cards

Reply:
Yes, the asic command worked on single asic.  Fixed in latest.